### ISSUE TITLE
initContainers not deployed when sysctlImage is disabled

### ISSUE THREAD

**Issue by felicijus** [bitnami/charts#20856]
### Name and Version

bitnami/elasticsearch:19.13.5

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. no root environment, disable sysctlImage
2. values.yaml
3. Camunda Helm Chart (should have no impact on this)

### Are you using any custom parameters or values?

[issue.txt](https://github.com/bitnami/charts/files/13308046/issue.txt)

### What is the expected behavior?

Deploy the initContainers... like described in the Helm Chart

```yaml
## @param initContainers Add additional init containers to the all elasticsearch node pod(s)
## ref: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
## e.g:
## initContainers:
##  - name: your-image-name
##    image: your-image
##    imagePullPolicy: Always
##    command: ['sh', '-c', 'echo "hello world"']
##
initContainers: []
```

### What do you see instead?

No initContainers get deployed when `.Values.sysctlImage.enabled` is set to false.

### Additional information

Reason:

[bitnami/elasticsearch/templates/master/statefulset.yaml](https://github.com/bitnami/charts/blob/main/bitnami/elasticsearch/templates/master/statefulset.yaml)

```yaml
{{- if or .Values.master.initContainers .Values.sysctlImage.enabled (and .Values.volumePermissions.enabled .Values.master.persistence.enabled) }}
      initContainers:
```
Should also check for ".Values.initContainers"

**Comment by carrodher** [comment]
Thank you for bringing this issue to our attention. We appreciate your involvement! If you're interested in contributing a solution, we welcome you to create a pull request. The Bitnami team is excited to review your submission and offer feedback. You can find the contributing guidelines [here](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md).

Your contribution will greatly benefit the community. Feel free to reach out if you have any questions or need assistance.

**Comment by felicijus** [comment]
Fixed with #24005.
Thank you @BadLiveware.
